mpt/impl/imap-mailbox/jcr/src/test/java/org/apache/james/mpt/imapmailbox/jcr/JcrMailboxTest.java

```diff
@@ -18,12 +18,9 @@
  ****************************************************************/
 package org.apache.james.mpt.imapmailbox.jcr;
 
-import org.apache.james.mpt.imapmailbox.AbstractMailboxTest;
-import org.apache.james.mpt.onami.test.annotation.GuiceModules;
 import org.junit.Ignore;
 
 @Ignore("JWC-130 : JCR mailbox does not correctly release resources + append problems")
```

Comment:
Could we introduce a JAMES ticket for this? JWC is LINAGORA internal ticketing...